src/test/java/org/folio/rest/impl/FileUploadServiceTest.java

```diff
@@ -131,6 +132,7 @@ public void shouldUploadFile_return200Status(TestContext context) throws IOExcep
     File uploadedFile = new File(uploadedFileDefinition.getSourcePath());
     assertTrue(FileUtils.contentEquals(fileToUpload, uploadedFile));
     assertEquals(uploadedFileDefinition.getJobExecutionId(),  jobExecutions.getJobExecutions().get(0).getId());
+    assertNotNull(jobExecutions.getJobExecutions().get(0).getHrId());
```

Comment:
Maybe it makes sense later to create the endpoint to get job item by id, to do not to retrieve the collection

Reply:
that is true. We'll have to create more endpoints in the future too